homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -54,7 +85,9 @@ async def async_setup_entry(
         assert unique_id
 
     async_add_entities(
-        [SwissPublicTransportSensor(coordinator, unique_id)],
+        SwissPublicTransportSensor(coordinator, description, unique_id)
+        for description, data in zip(SENSORS, coordinator.data)
+        if description.exists_fn(data)
```

Comment:
But it can exist in the future right? (For example busses that only drive on weekdays, after a restart in the weekend)

Reply:
That is a good point! So I would propose to create as many as connections are requests (currently 3 and variable in the future)

Reply:
It will stay a 3 fixed, a service call will take care of the rest.